jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql

```diff
@@ -72,7 +72,7 @@ create table SessionInfo (id number(19,0) not null, lastModificationDate timesta
 create table Task (id number(19,0) not null, archived number(5,0), allowedToDelegate varchar2(255 char), description varchar2(255 char), formName varchar2(255 char), name varchar2(255 char), priority number(10,0) not null, subTaskStrategy varchar2(255 char), subject varchar2(255 char), activationTime timestamp, createdOn timestamp, deploymentId varchar2(255 char), documentAccessType number(10,0), documentContentId number(19,0) not null, documentType varchar2(255 char), expirationTime timestamp, faultAccessType number(10,0), faultContentId number(19,0) not null, faultName varchar2(255 char), faultType varchar2(255 char), outputAccessType number(10,0), outputContentId number(19,0) not null, outputType varchar2(255 char), parentId number(19,0) not null, previousStatus number(10,0), processId varchar2(255 char), processInstanceId number(19,0) not null, processSessionId number(19,0) not null, skipable number(1,0) not null, status varchar2(255 char), workItemId number(19,0) not null, taskType varchar2(255 char), OPTLOCK number(10,0), taskInitiator_id varchar2(255 char), actualOwner_id varchar2(255 char), createdBy_id varchar2(255 char), primary key (id));
 create table task_comment (id number(19,0) not null, addedAt timestamp, text clob, addedBy_id varchar2(255 char), TaskData_Comments_Id number(19,0), primary key (id));
 create table TaskDef (id number(19,0) not null, name varchar2(255 char), priority number(10,0) not null, primary key (id));
-create table TaskEvent (id number(19,0) not null, logTime timestamp, message varchar2(255 char), processInstanceId number(19,0), taskId number(19,0), type varchar2(255 char), userId varchar2(255 char), OPTLOCK number(10,0), workItemId number(19,0), correlationKey varchar(255), processType number(1,0), primary key (id));
+create table TaskEvent (id number(19,0) not null, correlationKey varchar2(255 char), logTime timestamp, message varchar2(255 char), processInstanceId number(19,0), processType number(10,0), taskId number(19,0), type varchar2(255 char), userId varchar2(255 char), OPTLOCK number(10,0), workItemId number(19,0), primary key (id));
```

Comment:
processType is being changed from `number(1,0)` to `number(10,0)` - is this a typo?